superdesk/publish/subscriber_token.py

```diff
@@ -4,6 +4,7 @@
 from datetime import timedelta
 from superdesk.utc import utcnow
 from superdesk.utils import get_random_token
+from content_api import MONGO_PREFIX
```

Comment:
It's strange to see here import from `content_api`, seems we need to reorganize things regarding `superdesk`, `apps`, `content_api`